### ISSUE TITLE
New Demo UI

### ISSUE THREAD

**Issue by rui-cruz** [rui-cruz/Spark-Material#18]
Hi,

Components are growing day-by-day, and at this time the form is not big enough.. I think we need to prototype something new.

I'll create a new branch from #5 to create a new UI. But I think Accordion component is required first.

**Comment by olafkrueger** [comment]
Yes, also a good idea, There should be something like an component explorer.
From my point of view the accordion should behave like the "Angular Material" navigation.
I remember that the old flexlib project provides a componant called "WIndowShade" that behave similar to this. Perhaps it could be used as inspiration.

**Comment by rui-cruz** [comment]
What you think about this?

![prototype](https://cloud.githubusercontent.com/assets/1080374/10926711/b67035bc-8291-11e5-8a2a-04583c625e65.png)

**Comment by quick6black** [comment]
looks good.

**Comment by olafkrueger** [comment]
Looks promising. Could you provide the code including the "Navigation Component"?

**Comment by rui-cruz** [comment]
Sure, after I code it.. lol ^^ this is PhotoShop mockup xD

**Comment by olafkrueger** [comment]
:-)))))))))))))))))))))))))))))
